lib/views/help/officers.html.erb

```diff
@@ -1,245 +1,115 @@
-and/or give technical details of delivery so an IT department can chase
-up what happened to the message.
-</p>
-<p>Finally, you can respond to any request from your web browser, without
-needing any email, using the "respond to request" link at the bottom of
-each request page.
-</dd>
 
-<dt id="days">How do you calculate the deadline shown on request pages?<a href="#days">#</a> </dt>
+<dt id="responses">¿Por qué se publican las respuestas a las solicitudes de información? <a href="#responses">#</a> </dt>
 
 <dd>
-<p>The Freedom of Information Act says:</p>
+  <p>Creemos que aporta beneficios. Principalmente animará a los ciudadanos a estar más interesados y comprometidos con las actividades de las instituciones públicas.</p>
 
-<blockquote><p>A public authority must comply with section 1(1) <strong>promptly</strong> and
-in any event not later than the twentieth working day following the date of
-receipt.</p></blockquote>
-
-<p>The nerdy detail of exactly how weekends are counted, and what happens if
-the request arrives out of office hours, is just that - detail. What matters
-here is that the law says authorities must respond <strong>promptly</strong>.</p>
-
-<p>If you've got a good reason why the request is going to take a while to
-process, requesters find it really helpful if you can send a quick email with a
-sentence or two saying what is happening. </p>
-
-<p>FOI officers often have to do a lot of <strong>hard work</strong> to answer
-requests, and this is hidden from the public. We think it would help everyone
-to have more of that complexity visible.</p>
+  <p>También esperamos que, de esta manera, se reduzca el número de solicitudes de información duplicadas que puedan recibir organismos como el suyo.</p>
 
+  <p>La información que se va a solicitar es información pública por lo que su posterior publicación no supone ningún problema, al contrario: es una forma eficiente de que ésta no vuelva a ser solicitada.</p>
 </dd>
 
-<dt id="days2">But really, how do you calculate the deadline?<a href="#days2">#</a> </dt>
+<dt id="realpeople">¿Las personas que realizan solicitudes son reales? <a href="#realpeople">#</a> </dt>
 
 <dd>
+  <p>Por supuesto. Antes de poder enviar una solicitud, todos los usuarios deben registrarse en la página web aportando un e-mail que verificamos.</p>
 
-<p>Please read the answer to the previous question first. Legally, authorities
-must respond <strong>promptly</strong> to FOI requests. If they fail to do that,
-it is best if they show the hard work they are doing by explaining what is
-taking the extra time to do.
-</p>
-
-<p>That said, <%= site_name %> does show the maximum legal deadline
-for response on each request. Here's how we calculate it.</p>
-
-<ul>
+  <p>Para que el sistema funcione de manera automática, utilizamos direcciones de correo electrónico generadas por ordenador para cada una de las solicitudes. Es por eso que la dirección de contacto que a usted le aparece pone “@datasketch.co”.</p>
```

Comment:
I think I'll change that to `<%= AlaveteliConfiguration::incoming_email_domain %>` as per the belgium theme